doc/integrator/filter.rst

```diff
@@ -3,8 +3,19 @@
 Filter (Querier)
 ================
 
+Available attributes and operators in filters
+---------------------------------------------
+
+All attributes defined as "exported" in the layer of you map server will be automatically
```

Comment:
of your  mapserver

Reply:
That can be qgis, arcgis... a server of maps

Reply:
yes, but it is "of your map server" (not "of you map server")

Reply:
ah , ha, yes indeed